test/Microsoft.Azure.Devices.Edge.Test/Provisioning.cs

```diff
@@ -67,5 +74,55 @@ public async Task DpsSymmetricKey()
             Context.Current.DeleteList.TryAdd(registrationId, device.Expect(() => new InvalidOperationException(
                 $"Device '{registrationId}' should have been created by DPS, but was not found in '{this.iotHub.Hostname}'")));
         }
+
+        [Test]
+        public async Task DpsX509()
+        {
+            (string, string, string) rootCa =
+                        Context.Current.RootCaKeys.Expect(() => new InvalidOperationException("Missing root CA keys"));
+            string caCertScriptPath =
+                        Context.Current.CaCertScriptPath.Expect(() => new InvalidOperationException("Missing CA cert script path"));
+            string idScope = Context.Current.DpsIdScope.Expect(() => new InvalidOperationException("Missing DPS ID scope"));
+            string registrationId = this.GetRegistrationId();
+
+            Assert.IsTrue(File.Exists(Path.Combine(caCertScriptPath, "certGen.sh")), "CA Certification script does not exist. Path: " + $"{caCertScriptPath}" );
```

Comment:
This assert probably won't fail on Windows (because windows needs `ca-certs.ps1` which happens to live in the same directory), but it also doesn't make sense for Windows. The test shouldn't really care if that script exists because it's an implementation detail of `CertificateAuthority` and `LeafDevice`. If the script doesn't exist we should know it from the output of `Process.RunAsync`, so I'd remove just this.